config/config.go

```diff
@@ -35,10 +35,12 @@ type (
 		}
 
 		Agent struct {
-			Host        string
-			Token       string
-			Image       string `default:"drone/agent:0.8"`
-			Concurrency int    `default:"2"`
+			Host             string
+			Token            string
+			Image            string        `default:"drone/agent:0.8"`
+			Concurrency      int           `default:"2"`
+			KeepaliveTime    time.Duration `default:"6m"`
+			KeepaliveTimeout time.Duration `default:"30s"`
```

Comment:
note that when you declare these variables here, they are automatically being parsed from the environment when the system loads. In this case, the system will look for environment variables using `DRONE_AGENT_KEEPALIVETIME` and `DRONE_AGENT_KEEPLIVETIMEOUT` (no space). Note that you can customize the environment variable names. See some of the other attributes in the Config object for examples or refer to https://github.com/kelseyhightower/envconfig

Reply:
Thank you for all the feedback.
I was a bit puzzled how certain env vars did get parsed, your comments clear up a lot here.

I hope to have some time Thursday/Friday to apply your feedback.